mixins/harvester-vm/index.js

```diff
@@ -614,6 +622,52 @@ export default {
       this.$set(this.spec.template, 'spec', spec);
     },
 
+    parseAccessCredentials() {
+      if (this.resource !== HCI.VM || this.isCreate) {
+        return;
+      }
+
+      const out = [];
+      const annotations = {};
+      const users = JSON.parse(this.spec.template.metadata.annotations[HCI_ANNOTATIONS.DYNAMIC_SSHKEYS_USERS] || '[]');
+
+      for (const row of this.accessCredentials) {
+        if (!row.secretName) {
+          continue;
+        }
+
+        if (row.source === ACCESS_CREDENTIALS.RESET_PWD) {
+          users.push(row.username);
+          out.push({
+            userPassword: {
+              source:            { secret: { secretName: row.secretName } },
+              propagationMethod: { qemuGuestAgent: { users: [row.username] } }
```

Comment:
It should be useless `users: [row.username]`

Reply:
Done.